docs/Support-multiple-reference-genomes.md

```diff
@@ -0,0 +1,35 @@
+# [Introduction](introduction)
+From release 3.1, cBioportal supports multiple reference genomes. This means the hg19 datasets can co-exist with the hg38 ones in the same portal database. We also support mouse genomes (mm10), but it is not recommended to have both human and mouse data in the same instance of the cBioPortal.
+
+This document outlines the steps to support multiple reference genomes in your instance of cBioPortal.
+
+# [How is it supposed to work?](workflow)
+1. [Import Reference Genome](Import-reference-genome) into your cBioPortal database. If the version of your portal instance is lower than 3.1, you will have to [update your cBioPortal installation](Updating-your-cBioPortal-installation) 
+and migrate your database schema to the latest. The migrartion script by default will add three reference genomes (hg19, hg38, mm10) to the database.
+2. [Update the reference genome gene database table](Updating-gene-and-gene_alias-tables) to include the genes from the reference genome of interest to the database. You will also need to update both gene and gene alias tables in order to support other species such as mouse.
```

Comment:
If I start a new instance from scatch, do I need # 1&2 or are they included in the seed db?

If I migrate my db, will the migration script take care of both 1&2?

Reply:
if you are using a seed database to restore you database or using the migration script you will still need to do step 2 to add genes from your new reference genome.